HotelBase/ReservationOperationsProvider/ReservationOperationsProviderSeasonPrice.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
```

Comment:
Maybe I have missed something, but it looks like not used class.

Reply:
corrected